flocker/control/_config.py

```diff
@@ -1259,11 +1259,17 @@ def deployment_from_configuration(deployment_state, deployment_configuration,
     seen_applications = set()
     for hostname, application_names in (
             deployment_configuration['nodes'].items()):
+        # If application_names is a string type, convert to a list.
+        # This allows single a application name in a deployment configuration.
+        # The application will still be rejected if it does not exist in the
+        # application manifest.
+        if isinstance(application_names, (str, unicode)):
+            application_names = [application_names]
```

Comment:
So you just want the doc change? I don't have a personal preference, but I wonder why would we not want to accept a single app name as a string in the case of a deployment config only deploying one app? Forcing people to put a `-` or `[]` in the config for one application is possibly a more confusing experience for the user.